packages/bitcore-wallet-service/src/lib/chain/eth/index.ts

```diff
@@ -125,7 +125,12 @@ export class EthChain implements IChain {
               coin,
               network,
               from,
-              to: opts.tokenAddress || opts.multisigContractAddress || output.toAddress,
+              to:
+                opts.tokenAddress && !opts.multisigContractAddress
+                  ? opts.tokenAddress
+                  : opts.multisigContractAddress
+                  ? opts.multisigContractAddress
```

Comment:
My brain struggles with the double ternarys

Is this the same as:

`opts.tokenAddress || opts.multisigContractAddress || output.toAddress?`